OmiseSwift/Destroyable.swift

```diff
@@ -5,20 +5,20 @@ public protocol Destroyable { }
 public extension Destroyable where Self: ResourceObject {
     public typealias DestroyOperation = Operation<Self>
     
-    public static func destroyOperation(parent: ResourceObject?) -> DestroyOperation {
+    public static func destroyOperation(_ parent: ResourceObject?, id: String) -> DestroyOperation {
```

Comment:
It's ok. Your PR should be merged first then I will rebase on top of your PR again.